packages/discord.js/src/util/ActivityFlagsBitField.js

```diff
@@ -19,6 +19,7 @@ class ActivityFlagsBitField extends BitField {}
 /**
  * Numeric activity flags.
  * @type {ActivityFlags}
+ * @private
```

Comment:
```suggestion
 * @protected
```

Reply:
Does that render correctly on the docs website?